public/css/icinga/layout.less

```diff
@@ -333,6 +333,10 @@
     .nav-level-1 > .nav-item i {
       font-size: 1.5em;
       margin-right: .5em;
```

Comment:
This is actually related to the specificity of its rule. This margin is only relevant to level 1 nav items when the sidebar is collapsed. At the moment, level 2 nav items are also affected because they are contained and also have items with icons. The icon `.nav-item i` selector must be limited to only affect icons related to level 1 nav items.